hardhat.config.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+
+module.exports = {
+  solidity: '0.7.3',
+  networks: {
+    hardhat: {
+      forking: {
+        url: 'https://bsc-dataseed1.ninicoin.io',
+        blockNumber: 6771269,
```

Comment:
Why a hard set block number?

Reply:
In the [first attempt](https://github.com/pancakeswap/pancake-frontend/pull/376/commits/afa1e6fe5d981d754236b5c9755716b97d809e9b) to integrate Hardhat mainnet forking I saw an block number added.

Hardhat recommends branching from a specific block number to set up a fork-dependent test suite.

But it is completely optional.

what are your thoughts?

Reply:
I can see why. I wonder though if it can be configured on a case-by-case basis.